app/models/financialDetails/WhatYouOweChargesList.scala

```diff
@@ -64,15 +64,15 @@ case class WhatYouOweChargesList(balanceDetails: BalanceDetails, chargesList: Li
       if (bcdDueDate.isBefore(sortedListOfCharges.get.dueDate.get)) {
         Some((bcdDueDate.getYear, outstandingChargesModel.get.bcdChargeType.get.chargeAmount))
       } else if (sortedListOfCharges.isDefined) {
-          Some((sortedListOfCharges.get.dueDate.get.getYear, sortedListOfCharges.get.documentDetail.remainingToPayByChargeOrLpi))
+          Some((sortedListOfCharges.get.dueDate.get.getYear, sortedListOfCharges.get.documentDetail.remainingToPayByChargeOrInterest))
```

Comment:
remainingToPayByChargeOrInterest => remainingToPayByChargeOrLaterPaymentInterest  ~ (Lpi => Later payment Interest still needed I think to preserver this in naming convention )

Reply:
It's now not just late payment interest though, it's late payment interest OR this new kind of interest on reconciliation debits